endpoints/cookie_sync.go

```diff
@@ -455,11 +462,31 @@ func (c *cookieSyncEndpoint) handleResponse(w http.ResponseWriter, tf usersync.S
 	})
 
 	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+
 	enc := json.NewEncoder(w)
 	enc.SetEscapeHTML(false)
 	enc.Encode(response)
 }
 
+func (c *cookieSyncEndpoint) setCookieDeprecationHeader(w http.ResponseWriter, r *http.Request, account *config.Account) {
+	if rcd, err := r.Cookie(receiveCookieDeprecation); err == nil && rcd != nil {
+		return
+	}
+	if account == nil || !account.Privacy.PrivacySandbox.CookieDeprecation.Enabled {
+		return
+	}
+	http.SetCookie(w, &http.Cookie{
+		Name:     receiveCookieDeprecation,
+		Value:    "1",
+		Secure:   true,
+		HttpOnly: true,
+		Path:     "/",
+		SameSite: http.SameSiteNoneMode,
+		// Partition: "",
```

Comment:
yes, partition is required. How about we write partition manually until the change is added to Golang?. Ex. something like
```
func setCookie(w http.ResponseWriter, cookie *http.Cookie) {
	if v := cookie.String(); v != "" {
		w.Header().Add("Set-Cookie", v+" Partitioned;")
	}
}
```

Reply:
This seems like a reasonable approach. Can you call it from within `setCookieDeprecationHeader` and also keep this as a separate function properly annotated with a comment above the signature describing in just a few words that this is necessary because golang has yet to incorporate Partition support?

Reply:
done